runtime/centrifuge/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "centrifuge-runtime"
-version = "0.10.22"
```

Comment:
I understand this has not been changed for `altair` because we are using development as demo now.

Reply:
The correct reason would be that we had not upgraded Algol or Altair yet. In contrast, Catalyst received two upgrades already from 1020 (current Centrifuge version) to 1022.

Reply:
Thanks for extend!